src/Composer/Installers/DrupalInstaller.php

```diff
@@ -3,14 +3,17 @@
 
 class DrupalInstaller extends BaseInstaller
 {
-    protected $locations = array(
-        'core'      => 'core/',
-        'module'    => 'modules/{$name}/',
-        'theme'     => 'themes/{$name}/',
-        'library'   => 'libraries/{$name}/',
-        'profile'   => 'profiles/{$name}/',
-        'drush'     => 'drush/{$name}/',
-	    'custom-theme' => 'themes/custom/{$name}/',
-	    'custom-module' => 'modules/custom/{$name}',
-    );
+	protected $locations = array(
+		'core'                => 'core/',
```

Comment:
`drupal-core`, `drupal-module`, and `drupal-theme` were bold in the readme... Which it states they are stable and install paths will not be changed.